.github/workflows/build.yml

```diff
@@ -61,6 +61,10 @@ jobs:
           - os: "alpine"
             pkgs: "gcc"
             env1: "CC=gcc CXX=g++"
+          - os: "fedora:latest"
+            pkgs: "gcc gcc-g++"
+          # - os: "fedora:rawhide"
```

Comment:
do we need this comment?

Reply:
I just added it as a reminder and if anyone wanted to test it. It can be enabled in future.

Reply:
it should be either added to the matrix or removed from the config. There is no need to leave the dead code here.